frontend/libs/studio-components/src/components/StudioExpression/StudioExpressionContext.ts

```diff
@@ -0,0 +1,10 @@
+import { createContext } from 'react';
+import type { DataLookupOptions } from './types/DataLookupOptions';
+import type { ExpressionTexts } from './types/ExpressionTexts';
+
+export type StudioExpressionContext = {
+  dataLookupOptions: DataLookupOptions;
+  texts: ExpressionTexts;
+};
+
+export const StudioExpressionContext = createContext<StudioExpressionContext>(null);
```

Comment:
How about developing a custom hook to directly leverage the StudioExpressionContext? This way, we avoid importing it multiple times wherever it's needed. 😊 


```suggestion
const StudioExpressionContext = createContext<StudioExpressionContext>(null);
export const useStudioExpressionContext = useContext(StudioExpressionContext);
```

Reply:
I considered that. However, the change in implementation would be minimal: `useStudioExpressionContext()` vs. `useContext(StudioExpression)`. What I like about the current approach is that the developer can navigate directly to the context without going through an additional layer. But I don't have any strong opinions about this. I'll change it 🙂